README.md

```diff
@@ -85,6 +85,7 @@ require("dapui").setup({
   floating = {
     max_height = nil, -- These can be integers or a float between 0 and 1.
     max_width = nil, -- Floats will be treated as percentage of your screen.
+	border = "single", -- Border style. Can be "single", "double" or "rounded"
```

Comment:
You're using tabs rather than spaces here and in the config, please swap to spaces